project.json

```diff
@@ -13,8 +14,14 @@
 		"Microsoft.AspNetCore.Server.Kestrel": "1.0.1",
 		"Microsoft.AspNetCore.StaticFiles": "1.0.0",
 		"Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0",
-		"Microsoft.Extensions.Logging.Console": "1.0.0"
-	},
+    "Microsoft.Extensions.Logging.Console": "1.0.0",
+    "Microsoft.AspNetCore.Mvc": "1.0.1",
+    "Microsoft.AspNetCore.Routing": "1.0.1",
+    "Microsoft.AspNetCore.Diagnostics": "1.0.0",
+    "Microsoft.VisualStudio.Web.BrowserLink.Loader": "14.0.0",
+    "Microsoft.AspNetCore.Razor.Tools": "1.0.0-preview2-final",
+    "Microsoft.Extensions.Configuration.FileExtensions": "1.0.0",
+    "Microsoft.Extensions.Configuration.Json": "1.0.0"	},
```

Comment:
These two extensions shouldn't be necessary (`Json` and `FileExtensions`).  Similarly, I don't think `BrowserLink.Loader` is necessary nor is `Diagnostics`.  Recommend removing them and see if everything still works.

I'm not sure what `Razor.Tools` is for, I have a sneaking suspicion it isn't necessary either but I'm less confident.  Can you try removing it and seeing if things still work?